Applications/AdminUi/packages/admin_api_types/lib/src/relationships/relationship.dart

```diff
@@ -6,7 +6,7 @@ part 'relationship.g.dart';
 class Relationship {
   final String peer;
   final String requestedBy;
-  final String templateId;
+  final String? templateId;
```

Comment:
Why is the template id optional now?

Reply:
It is so for quite a while. We just never updated the type definitions in the flutter app.

The reason why it is optional is that the relationship template might get deleted.